subprojects/gradle/instrumentation-modified-tests-finder/src/main/kotlin/com/avito/android/InstrumentationModifiedTestsFinderPlugin.kt

```diff
@@ -0,0 +1,17 @@
+package com.avito.android
+
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import org.gradle.kotlin.dsl.create
+import org.gradle.kotlin.dsl.register
+
+class InstrumentationModifiedTestsFinderPlugin : Plugin<Project> {
```

Comment:
Why this finder is not a part of the instrumentation plugin?

Reply:
I see value in the additional module but don't understand the plugin.

Reply:
not much, but i can test with test kit in separate module now

Reply:
i want to left it as is for now

Reply:
Additional plugin impacts user experience if it's not independent.
Let's schedule a discussion. I'd like to define alternatives and agree about common conventions.